docs/user_guide/assessment.rst

```diff
@@ -216,6 +216,56 @@ case we see that the subgroup ``(a, 8)`` has a result of ``NaN``, indicating
 that there were no samples in it.
 
 
+Scalar Results from :code:`MetricFrame`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Higher level machine learning algorithms (such as hyperparameter tuners) often
+make use of metric functions to guide their optimisations.
+Such algorithms generally work with scalar results, so if we want the tuning
+to be done on the basis of our fairness metrics, we need to perform aggregations
+over the :class:`.MetricFrame`.
+
+We provide a convenience function, :func:`fairlearn.metrics.make_derived_metric`
+to generate scalar-producing metric functions based on the aggregation methods
+mentioned above (:meth:`.MetricFrame.group_min`, :meth:`.MetricFrame.group_max`,
+:meth:`.MetricFrame.difference`, and :meth:`.MetricFrame.ratio`).
+This takes an underlying metric function, the name of the desired transformation, and
+optionally a list of parameter names which should be treated as sample parameters.
+The result is a function internally builds the :class:`.MetricFrame` and performs
+the requested aggregation. For example:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.metrics import make_derived_metric
+    >>> fbeta_difference = make_derived_metric(metric=skm.fbeta_score,
+    ...                                        transform='difference')
+    >>> fbeta_difference(Y_true, Y_pred, beta=0.7,
+    ...                  sensitive_features=group_membership_data)
+    0.7525252525...
+    >>> fbeta_07 = functools.partial(skm.fbeta_score, beta=0.7)
+    >>> MetricFrame(fbeta_07,
+    ...             Y_true, Y_pred,
+    ...             sensitive_features=group_membership_data).difference()
+    0.7525252525...
+
+We use :func:`fairlearn.metrics.make_derived_metric` to manufacture a number
+of such functions which will be commonly used:
+
+=============================================== ================= ================= ================== =============
+Base metric                                     :code:`group_min` :code:`group_max` :code:`difference` :code:`ratio`
```

Comment:
I love the table! Small suggestions
* Rather than "Y/N" in the table, can we do "Y" and "-"? Or if you want to be fancy, you could use &#x2713; for "Y". But "-" instead of "N" is nicer, because it makes it much easier to tell what's supported and what's not.
* [this is a very low priority--don't spend more than 5 mins on this] Does sphinx link to the correct content if we only write
```
:func:`.true_positive_rate`
:func:`.accuracy_score`
```
Or possibly something like `flm.true_positive_rate` and `skm.precision_score`? (Not sure if sphinx can deal with named imports.)

Reply:
Good point about the relative visual impact of 'Y' and '-'. However, since `-` starts a bullet list in ReST, I've swapped to a '.' instead.

For the links, I can do the short version for Fairlearn but the external links aren't working. There's probably a way, based on what is 'in scope' for sphinx at the time, but some searching around hasn't turned up a straightforward way of achieving it.